models/Post.model.js

```diff
@@ -0,0 +1,16 @@
+const { Schema, model } = require('mongoose');
+ 
+const postSchema = new Schema(
+  {
+    user: [{ type: Schema.Types.ObjectId, ref: 'User' }],
```

Comment:
Hey @ConjecturesandRefutations,  we need to remove this line because this means that a post can have multiple users. It's a thing that we don't want :)